src/ci/docker/scripts/emscripten.sh

```diff
@@ -40,9 +40,12 @@ hide_output ./emsdk install sdk-1.37.13-64bit
 source ./emsdk_env.sh
 echo "main(){}" > a.c
 HOME=/emsdk-portable/ emcc a.c
-HOME=/emsdk-portable/ emcc -s BINARYEN=1 a.c
+HOME=/emsdk-portable/ emcc -s WASM=1 a.c
 rm -f a.*
 
+# Make emscripten use Rust's LLVM
```

Comment:
Mind expanding the comment here a bit? I'm actually not even sure what it means for emscripten to use someone else's llvm, isn't it precompiled?